Makefile.helpers

```diff
@@ -26,18 +26,19 @@ endef
 default:: $(DEFAULT_HELP_TARGET)
 	@exit 0
 
-.PHONY : help
-## This help screen
-help:
```

Comment:
Unsure how I feel about this. We show `make help` everywhere in docs, `README.md`, etc. Now it's gone and won't work. I think it should be replaced with:

```
.PHONY : help
## Default help
help:: $(DEFAULT_HELP_TARGET)
  @exit 0
```

Reply:
If we will add `help` target then 
we can not do this way
https://github.com/sportech-dev/local-dev/blob/chamber-support/Makefile#L21

Reply:
Looks like I have idea.